templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1026,14 +1039,54 @@ Resources:
         Statement:
           - Effect: "Allow"
             Principal:
-              AWS: !GetAtt BitbucketClusterNodeRole.Arn
-            Action: "es:*"
+              AWS: "*"
+            Action: "es:ESHttp*"
             Resource: "*"
+      AdvancedSecurityOptions:
+        Enabled: true
+        InternalUserDatabaseEnabled: true
+        MasterUserOptions:
+          MasterUserName: bitbucket
+          MasterUserPassword: !Ref ElasticSearchPassword
+      DomainEndpointOptions:
+        EnforceHTTPS: true
+        # Should we enforce TLS 1.2?
+        TLSSecurityPolicy: Policy-Min-TLS-1-2-2019-07
+      EncryptionAtRestOptions:
+        Enabled: true
+      NodeToNodeEncryptionOptions:
+        Enabled: true
+      # VPCOptions:
```

Comment:
This is the configuration required to put the ES cluster in the VPC (and make it private to public internet), however, I couldn't get it to work today.

Reply:
Update, I got it working by adding the security group